context/package.json

```diff
@@ -7,13 +7,17 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.45",
     "fromentries": "^1.2.0",
+    "lodash": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-twitter-embed": "^3.0.3",
+    "react-vega": "^7.3.0",
     "searchkit": "^2.4.1-alpha.4",
     "styled-components": "^5.1.0",
     "typeface-inter": "^3.12.0",
+    "vega": "^5.13.0",
+    "vega-lite": "^4.13.1",
```

Comment:
Do these need to be here, or could NPM handle the transitive dependencies? I don't see them being explicitly imported in the code.

Reply:
They are peer dependencies of `react-vega`